### ISSUE TITLE
v3.1 button does not show up in Vivaldi browser

### ISSUE THREAD

**Issue by ericblade** [natalieperna/layout-trello#29]
Vivaldi is a Chrome based browser, and I've had no trouble with about a dozen other extensions, but I cannot get List Layouts for Trello to appear in the bar at all :(

**Comment by spellitwithaph** [comment]
Hi @ericblade, this sounds similar to https://github.com/natalieperna/layout-trello/issues/25. Can you please have a look at this issue and see if it solves your problem? I've had no issues with this plugin in either Vivaldi or Opera (both Chromium based). 

If this doesn't resolve your issue can you please provide more feedback about your environment (OS/version, Browser/version, screenshots, etc) to help with some investigation. Thanks!

**Comment by ericblade** [comment]
oooh, thanks for the tip.  I'll try to take some time today to see if I can fix it that way, and although I've already reported that it doesn't work to the Vivaldi team, I'll definitely add a link to that "solution" to my bug report with them.  Thanks!

**Comment by ericblade** [comment]
OK, so, this is kind of hilarious --

 In Vivaldi, site-specific extension buttons appear in the address bar, when you are on those sites.  Non-specific appear to the right of the address bar, as they do on Chrome.

So, the Trello icon *is* showing up for me, it's just not in any place I expected it to be, so I never saw it.

I only saw it after I installed the Stable Vivaldi, and had zero extensions .. then I installed this extension, and noticed the new icon on the page after loading trello.com.  So I checked in the development version, and sure enough, there it is.

I have to go retract my bug report from Vivaldi now too because I wasn't expecting it to be in the address bar.

Works great, by the way! Thank you for the extension!  If this didn't exist, I'd have to write one myself, because Trello is unusable with more than a few cards, without this.
